ldk/csharp/OliveHelpsLDK/OliveHelpsLDK/ExceptionLoggingInterceptor.cs

```diff
@@ -162,7 +162,7 @@ private static void HandleException(AggregateException aex, ILogger logger, Canc
                 aex.Handle(exception =>
                 {
                     logger.Error("Client exception", exception);
-                    return true;
+                    return false;
```

Comment:
This makes 0 difference to everything BUT the duplex streams - it mostly means that the TaskScheduler will re-throw the exception, which is the desired outcome.